src/components/NotesPage/NotesPageTopBar.js

```diff
@@ -16,34 +18,60 @@ import { LogoutButtonContainer } from 'components/LogoutButton/LogoutButtonConta
 
 import * as Selectors from 'components/Uploads/UploadsSelectors';
 
+import { colors } from 'colors';
+
 export const NotesPageTopBar = () => {
   const hasUploads = useSelector(Selectors.hasUploads);
   // const hasActiveUploads = useSelector(Selectors.hasActiveUploads);
 
+  const [state, setState] = React.useState(0);
+
+  const handleClick = () => {
+    setState(60);
+  };
+
+  const handleClose = () => {
+    setState(0);
+  };
+
   return (
     <TopBar>
       <TopBarLeftButtons>
-        <IconButton edge="start" color="inherit" aria-label="menu">
-          <MenuIcon />
+        <IconButton
+          onClick={handleClick}
+          edge="start"
+          color="inherit"
+          aria-controls="burger-menu">
+          <MenuIcon color="primary" />
         </IconButton>
+        <Menu
+          id="burger-menu"
+          anchorReference="anchorPosition"
+          anchorPosition={state ? { top: state, left: 0 } : undefined}
+          open={!!state}
```

Comment:
Please rename `state` to something more meaningful.

If you're using it for `open` property of the `Menu`, name it `isMenuOpen`.